This is easier than getting earlier versions to build.vcdimager's manpage didn't
change at 2.0, so I hope its output is the same and still understandable by
Brasero. I can't test this, I do not have any VCDs!

---
diff --git a/plugins/vcdimager/burn-vcdimager.c b/plugins/vcdimager/burn-vcdimager.c
index 7cafd7a2..54284bd6 100644
--- a/plugins/vcdimager/burn-vcdimager.c
+++ b/plugins/vcdimager/burn-vcdimager.c
@@ -525,7 +525,7 @@ brasero_vcd_imager_export_caps (BraseroPlugin *plugin)
 G_MODULE_EXPORT void
 brasero_plugin_check_config (BraseroPlugin *plugin)
 {
-	gint version [3] = { 0, 7, 0};
+	gint version [3] = { 2, 0, 0};
 	brasero_plugin_test_app (plugin,
 	                         "vcdimager",
 	                         "--version",
